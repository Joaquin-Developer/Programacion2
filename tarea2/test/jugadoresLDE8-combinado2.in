crearListaJugadores

crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 1 18 Uruguay Persona1
crearFecha 18/4/2023
insertarJugadorListaJugadores


crearJugada
1 4 3 2 1
agregarEnPartida
crearJugador 2 20 Uruguay Persona2
crearFecha 10/3/2023
insertarJugadorListaJugadores

crearJugada
1 1 1 3 4
agregarEnPartida
crearJugador 3 25 Uruguay Persona3
crearFecha 22/4/2023
insertarJugadorListaJugadores

crearJugada
1 1 1 3 4
agregarEnPartida
crearJugada
2 1 1 2 2
agregarEnPartida
crearJugador 4 30 Brasil Persona4
crearFecha 12/11/2022
insertarJugadorListaJugadores


imprimirCantidadListaJugadores
imprimirListaJugadores

estaEnListaJugadores 5
estaEnListaJugadores 2

eliminarInicioListaJugadores

estaEnListaJugadores 3

imprimirMenorAMayorTJugadoresLDE

unirConVacioListaJugadores

imprimirMayorAMenorTJugadoresLDE

imprimirJugadorListaJugadores 1 
imprimirJugadorListaJugadores 4

liberarListaJugadores

Fin
