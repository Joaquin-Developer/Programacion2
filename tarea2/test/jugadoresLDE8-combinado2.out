1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>21>22>23>24>La cantidad de jugadores en la lista es: 4.
25>Comando no reconocido.
26>La jugador con id 5 NO está en la lista.
27>La jugador con id 2 está en la lista.
28>29>La jugador con id 3 NO está en la lista.
30>Jugador 4:  Persona4, Brasil, 30 años
Jugada: 1 
Movimientos: Derecha Derecha Arriba Abajo 
Jugada: 2 
Movimientos: Derecha Derecha Izquierda Izquierda 
12/11/2022
Jugador 2:  Persona2, Uruguay, 20 años
Jugada: 1 
Movimientos: Abajo Arriba Izquierda Derecha 
10/3/2023
Jugador 1:  Persona1, Uruguay, 18 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
18/4/2023
31>32>Jugador 1:  Persona1, Uruguay, 18 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
18/4/2023
Jugador 2:  Persona2, Uruguay, 20 años
Jugada: 1 
Movimientos: Abajo Arriba Izquierda Derecha 
10/3/2023
Jugador 4:  Persona4, Brasil, 30 años
Jugada: 1 
Movimientos: Derecha Derecha Arriba Abajo 
Jugada: 2 
Movimientos: Derecha Derecha Izquierda Izquierda 
12/11/2022
33>Jugador 1:  Persona1, Uruguay, 18 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
34>Jugador 4:  Persona4, Brasil, 30 años
Jugada: 1 
Movimientos: Derecha Derecha Arriba Abajo 
Jugada: 2 
Movimientos: Derecha Derecha Izquierda Izquierda 
35>36>Fin.
